data/transactions/logic/assembler.go

```diff
@@ -1073,6 +1104,11 @@ func (ops *OpStream) assemble(fin io.Reader) error {
 		}
 	}
 
+	// if the version was not fixed above (because there were no opcodes), fix it now
+	if ops.Version == assemblerNoVersion {
+		ops.Version = AssemblerDefaultVersion
+	}
+
```

Comment:
I only added this code because my optimization functions use `OpsByName[ops.Version]`, and some of the tests in `TestPragmas` broke this because they don't include any ops (meaning `ops.Version` remains -1). I don't expect this code to have any effect in actual programs.

Reply:
My latest commit removes this because it's no longer necessary.